src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1420,7 +1442,8 @@ static LfNetworkParameters getNetworkParameters(LoadFlowParameters parameters, O
                 .setPhaseControl(parameters.isPhaseShifterRegulationOn())
                 .setTransformerVoltageControl(parameters.isTransformerVoltageControlOn())
                 .setVoltagePerReactivePowerControl(parametersExt.isVoltagePerReactivePowerControl())
-                .setReactivePowerRemoteControl(parametersExt.hasReactivePowerRemoteControl())
+                .setGeneratorReactivePowerRemoteControl(parametersExt.hasGeneratorReactivePowerRemoteControl())
+                .setTransformerReactivePowerControl(parametersExt.isTransformerReactivePowerControl())
```

Comment:
it was like this before already, but I don't know why sometimes we use `has` and sometimes `is`

Reply:
unsure of what to do, if to be done would be done in another PR, resolving